app/workers/badge_achievements/badge_award_worker.rb

```diff
@@ -5,10 +5,14 @@ class BadgeAwardWorker
     sidekiq_options queue: :high_priority, retry: 10
 
     def perform(usernames, badge_slug, message)
```

Comment:
@mac Can you please help me verify how we award badges? Is most of it done through Heroku scheduler?

1. The first part of this `if` will take care of things that don't need extra arguments (yearly club, tags, 4, 8, and 16 week streak).
2. The second part here directly awards badges, what are we using this for?
3. The Rake tasks awards top seven, fab five, and contributor, how are these scheduled?
4. What about` award_contributor_badges_from_github` (now `Badges::AwardContributorFromGithub`)? This doesn't seem to be covered by any of these cases, are we still using it?

Reply:
@citizen428 Sorry I missed this. We aren't using Heroku Scheduler anymore. Instead, we are using `sidekiq-cron` gem (shoutout to @mstruve !). You can see each individual task schedule in `config/schedule.yml`.
2. I'm not sure quite sure either. I believe it for one-off awards. cc @benhalpern
3. & 4. You can see each individual task schedule in `config/schedule.yml`.

Reply:
Thanks @maestromac. I had actually looked for a file like this, no idea how I missed it 🤷